.changeset/fifty-nails-swim.md

```diff
@@ -0,0 +1,7 @@
+---
+'eslint-config-skuba': patch
+---
+
+Remove duplicate `@typescript-eslint` definitions
+
+This plugin is already defined in `eslint-config-seek/base` 14.
```

Comment:
```suggestion

```

Reply:
```suggestion
---
'eslint-config-skuba': patch
---

Remove duplicate `@typescript-eslint` definitions
```

I'm thinking we keep a brief mention to this because it's an issue with the current release version if you bump the underlying `typescript-eslint` version.